wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
Maybe you could add the reference to the origin of this script

Reply:
Origin of the script https://github.com/VirtoCommerce/vc-docker/blob/master/linux/platform/wait-for-it.sh

Reply:
Could you add it as a comment at the beginning of the script?